testsuite/tests/lib-arg/testerror.ml

```diff
@@ -0,0 +1,41 @@
+(** Test that the right message errors are emitted by Arg *)
+
+
+let usage= "Arg module testing"
+
+let test total i (spec,anon,argv) =
+  let argv = Array.of_list ("testerror" :: argv) in
+  try Arg.parse_argv ~current:(ref 0) argv spec anon usage with
+  | Arg.Bad s-> Printf.printf "(%d/%d) Bad:%s\n" (i+1) total s
+  | Arg.Help s -> Printf.printf "(%d/%d) Help:%s\n" (i+1) total s
```

Comment:
I think that `Help:\n%s\n` (and similarly for `Bad`) would make the reference file easier to read.